### DISCUSSION TITLE
R ospsuite.reportingengine Qualification Framework

### DISCUSSION THREAD

**Discussion by kcollins010** [Open-Systems-Pharmacology/Forum#969] [Questions & Problems] [answered]
Is there a qualification framework available for R ospsuite.reportingengine similar to the one for Matlab here?
https://docs.open-systems-pharmacology.org/shared-tools-and-example-workflows/qualification

SimulationSet() and MeanModelWorkflow() functions generate reports with good success, but I am also interested in generating tables for model building blocks / input parameters.  

Thanks,
Keagan

**Comment by msevestre** [comment, marked as answer]
We are actively working on it. We expect an R compatible release in the next couple of months. In the meantime, you can use the current code to try the current implementation.  Not all features from the Matlab implementation are available yet

**Reply by kcollins010** [reply]
Thanks for the reply, Michael.  I will work with the current code in the meantime.
